library/Director/Web/Form/PropertyTableSortForm.php

```diff
@@ -28,7 +26,38 @@ public function __construct(string $name, ValidHtml $table)
     protected function assemble()
     {
         $this->addElement('hidden', '__FORM_NAME', ['value' => $this->name]);
-        $this->addElement($this->createCsrfCounterMeasure(Session::getSession()->getId()));
+        $this->addElement($this->createCsrfCounterMeasure());
         $this->addHtml($this->table);
     }
+
+    /**
+     * Create a form element to countermeasure CSRF attacks
+     *
+     * @return FormElement
+     */
+    protected function createCsrfCounterMeasure()
```

Comment:
The return type for the method is missing.

Reply:
added